engine/src/main/java/org/apache/apex/engine/util/CascadeStorageAgent.java

```diff
@@ -16,17 +16,19 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.common.util;
+package org.apache.apex.engine.util;
```

Comment:
I guess it is mainly for internal consumption, better to move it to internal like you suggested.